src/documents/DeprecatedIn2_0.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Deprecated in 2.0
+layout: default
+category: Infusion
+---
+
+This page contains a list of the features, APIs, and etc. that are deprecated as of Infusion 2.0.
+
+<div class="infusion-docs-note"><strong>Note:</strong> While the intention is to provide advanced notice of future changes, the contents of this page may not be exhaustive.</div>
```

Comment:
Properly this should be "advance notice": http://grammarist.com/usage/advance-advanced/